src/sql/workbench/services/accountManagement/browser/accountPickerService.ts

```diff
@@ -48,7 +52,7 @@ export class AccountPickerService implements IAccountPickerService {
 	/**
 	 * Render account picker
 	 */
-	public renderAccountPicker(container: HTMLElement): void {
+	public renderAccountPicker(accountContainer: HTMLElement, tenantContainer): void {
 		if (!this._accountPicker) {
 			// TODO: expand support to multiple providers
 			const providerId: string = 'azure_publicCloud';
```

Comment:
This is interesting - how are people with non-public cloud stuff able to sign in then?

Reply:
They don't at the moment